js/jquery.dvfCharts.js

```diff
@@ -280,6 +281,48 @@
       return this;
     },
 
+    parseGaugeOptions: function () {
```

Comment:
I know this is just following suit with Jorges zero comments in Javascript :P but I feel every method should have at least one comment describing its purpose and in cases like this a link to the c3js reference.

eg 
```
/**
 * Parse user settings into object suitable for use with Gauge Chart.
 *
 * @example https://c3js.org/samples/chart_gauge.html
 * @see https://c3js.org/reference.html#gauge-label-show
 */
parseGaugeOptions: function () {
  ...
}
```